modules/deployment/src/test/resources/classloading-troubleshooting/errors/app-jms-properties-resource-not-found

```diff
@@ -1,6 +1,4 @@
-Failed to deploy artifact [app-classloading-troubleshooting-1.0.0-mule-application]
-org.mule.runtime.deployment.model.api.DeploymentException: Failed to deploy artifact [app-classloading-troubleshooting-1.0.0-mule-application]
-Caused by: org.mule.runtime.api.exception.MuleRuntimeException: org.mule.runtime.deployment.model.api.DeploymentInitException: ArtifactClassLoaderResourceNotFoundException: Couldn't find resource: jms.properties neither on classpath or in file system 
+Couldn't find resource: jms.properties neither on classpath or in file system 
```

Comment:
Unfortunately, due to the way in which slf4j-test saves logging events, it's not possible to assert the stack trace the same way it was done before. Log4j2 has a way of serializing the logging event so that the stack trace is saved and can, therefore, be later compared to this file and asserted. On the other hand, slf4j-test saves the logs on different events and all these events need to be asserted individually instead of all together against this file. That's why I had to create two different asserts and reduce the content of these comparing files.

Reply:
Agreed, Log4J appenders have access to the layout patters to use a certain format, when using the test lib we don't have any of that and it doesn't make sense to simulate it.